### DISCUSSION TITLE
Licensing and "as a service" offering

### DISCUSSION THREAD

**Discussion by kishaningithub** [trinodb/trino#11923] [Q&A] [answered]
From what i see this project uses Apache 2 license.. if you have been following whats happening around many projects have shifted to either a closed license or and AGPL 2.0 license because they dont want cloud providers to offer the work "as a service" without any/much patnership what soever

My concern is, I dont want the community to become unbalanced/split over it/creating separate projects as a result of that. So starting this discussion thread.

**Comment by martint** [comment, marked as answer]
Trino is an independent project and there are no intentions on changing the license from Apache 2. Of course, being an open source project with a liberal license, anyone can fork it and create/run their own version.
